cluster/gce/gci/configure-helper.sh

```diff
@@ -92,8 +94,10 @@ EOF
     missingok
     notifempty
     compress
-    size 100M
+    maxsize 100M
     daily
+    dateext
+    dateformat -%Y%m%d-%s
```

Comment:
please add a comment about the significance (i.e collisions, and that exceeding maxsize within a second will not rotate). A meta comment about how we expect logorate to happen will also help, something like: "logrotate daily or if the size exceeds maxsize, into a gzipped timestamped backup. Backups are discarded after <rotate>."